### DISCUSSION TITLE
Please can you add support to arch linux?

### DISCUSSION THREAD

**Discussion by p4rzl** [vilhelmprytz/pterodactyl-installer#505] [Ideas]
By giving the ability to install pterodactyl on arch, it would be easier for everyone to update and have continued support.

**Comment by sinjs** [comment]
Sadly [pterodactyl does not support Arch Linux](https://pterodactyl.io/panel/1.0/getting_started.html#picking-a-server-os), which is why the script will not introduce support for it.

**Comment by Linux123123** [comment]
Arch will not be supported.
